sed/config/NXmpes_config-HEXTOF.json

```diff
@@ -54,34 +55,32 @@
       "ky",
       "E"
     ],
-    "slow_axes": "@attrs:metadata/instrument/analyzer/slow_axes",
+    "slow_axes": "t.b.d.",
     "energy_resolution": {
-      "resolution": "@attrs:metadata/instrument/analyzer/energy_resolution",
+      "resolution": 100,
```

Comment:
Can these be entered into SciCat, or obtained in some other means?

Reply:
Yes, this can go into SciCat and was already added into ingesting script during last beamtime

Reply:
I see. Then maybe add a corresponding link here, or add manually.